tests/TestCase/Core/ConfigureTest.php

```diff
@@ -69,6 +69,26 @@ public function tearDown()
     }
 
     /**
+     * testGet method
+     *
+     * @return void
+     */
+    public function testGet()
+    {
+        $expected = 'ok';
+        Configure::write('This.Key.Exists', $expected);
+        $result = Configure::get('This.Key.Exists');
+        $this->assertEquals($expected, $result);
+
+        try {
```

Comment:
Because then you don't need try catch.
It will be a simple one-liner.
And one shouldnt mix positive and exception case for testing IMO.